src/inverseRobot.js

```diff
@@ -24,7 +24,22 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const res = { };
+
+  for (const key in robot) {
+    res[robot[key]] = key;
```

Comment:
```suggestion
    if (robot[key] in res) {
      return null;
    }
    res[robot[key]] = key;
```
This approach is way more simple and understandable.